### PR TITLE
More expression conversions 6

### PR DESCRIPTION
**Pull request by mmatera**
and more conversions.

<a href="https://gitpod.io/#https://github.com/Mathics3/mathics-core/pull/385"></a>

### CODE REVIEW

File: mathics/builtin/box/graphics.py [hunk 1]
```diff
@@ -114,7 +115,7 @@ def init(self, graphics, style, item):
                 else:
                     self.arc = (start_angle, end_angle)
 
-            item = Expression(item.get_head_name(), *item.elements[:2])
+            item = Expression(Symbol(item.get_head_name()), *item.elements[:2])
```
Comment (rocky) [inline mathics/builtin/box/graphics.py:118]: BTW - Symbol(element.get_head_name()) comes up often enough that we probably should arrange all simple or non-compound elements to have their symbols stored. 

I am sure there is a way to do this automatically for things like those things derived off of Bultin, but I don't know off hand. 

One of the many things for down the line.

File: mathics/core/systemsymbols.py [hunk 2]
```diff
@@ -146,4 +146,5 @@
 SymbolUndefined = Symbol("Undefined")
 SymbolUnequal = Symbol("Unequal")
 SymbolUnevaluated = Symbol("Unevaluated")
+SymbolUpset = Symbol("UpSet")
```
Comment (rocky) [inline mathics/core/systemsymbols.py:149]: This was defined in this commit in `symbols.py`

**Review by rocky — APPROVED** [review]
LGTM -  see comments

File: mathics/builtin/distance/numeric.py [hunk 3]
```diff
@@ -22,7 +23,7 @@
 
 
 def _norm_calc(head, u, v, evaluation):
```
Comment (rocky) [inline mathics/builtin/distance/numeric.py:25]: Tests seem to indicate that head is sometimes a symbol and sometimes a string. 

One reason I like to have type annotations in there (e.g. for `head`).

Comment (mmatera) [reply mathics/builtin/distance/numeric.py:25]: Yep... working on it.

**Comment by rocky** [comment]
@mmatera sorry about the black conflict if you rebase off of master, I think this will go away.

**Comment by rocky** [comment]
@mmatera sigh - I missed some test files (and black has changed its mind about these since 21.x. I find this kind of thing annoying, but supposedly black says it will stop changing its mind.)

So git merge and make sure to squash commits since all of this stuff is of little interest to anyone.

**Comment by mmatera** [comment]
> @mmatera sigh - I missed some test files (and black has changed its mind about these since 21.x. I find this kind of thing annoying, but supposedly black says it will stop changing its mind.)
> 
> So git merge and make sure to squash commits since all of this stuff is of little interest to anyone.

I made a rebase, so all the garbage should be gone...

**Comment by rocky** [comment]
> > @mmatera sigh - I missed some test files (and black has changed its mind about these since 21.x. I find this kind of thing annoying, but supposedly black says it will stop changing its mind.)
> > So git merge and make sure to squash commits since all of this stuff is of little interest to anyone.
> 
> I made a rebase, so all the garbage should be gone...

Thanks and thanks for pushing forward. Sorry about the black hassle. (We were at a beta version before. Hopefully this kind of crap we won't cause incompatibilities in the future.)

**Comment by mmatera** [comment]
> > > @mmatera sigh - I missed some test files (and black has changed its mind about these since 21.x. I find this kind of thing annoying, but supposedly black says it will stop changing its mind.)
> > > So git merge and make sure to squash commits since all of this stuff is of little interest to anyone.
> > 
> > 
> > I made a rebase, so all the garbage should be gone...
> 
> Thanks and thanks for pushing forward. Sorry about the black hassle. (We were at a beta version before. Hopefully this kind of crap we won't cause incompatibilities in the future.)

No problem, and thanks for fixing autoblack.

**Pull request merged**